modules/developers-guide/examples/javascript/node-processing1.js

```diff
@@ -0,0 +1,15 @@
+// Get the results from the execute query statement
+// and separate into an array to print out the results
+const resultSet = response.getResultSet();
```

Comment:
We should wrap everything after this line in an

```
if(resultSet) {

}
```

Because there's a chance the result set could be undefined. (I wish there wasn't, but that's generated code for ya...)

Reply:
ok

Reply:
changed all that used this